### ISSUE TITLE
/mining/candidate returns 500

### ISSUE THREAD

**Issue by neal-zhu** [ergoplatform/ergo#1309]
Hi, we call /mining/candidate api every 50ms, and we get some error response like:
```
{\n  \"error\" : 500,\n  \"reason\" : \"internal.error\",\n  \"detail\" : \"Skipping candidate generation, one is already in progress\"\n}
{\n  \"error\" : 500,\n  \"reason\" : \"internal.error\",\n  \"detail\" : \"Can not generate block candidate: chain not synced (maybe last block not fully applied yet\"\n}
```
And we found our node sometime does not sync with main-net which cause many orphan blocks.
Could you guys help us with this?

**Comment by pragmaxim** [comment]
@neal-zhu this should be fixed in the release after `4.0.10` ... I will update this issue when it is

**Comment by pragmaxim** [comment]
@neal-zhu hi, it addressed in 4.0.13

**Comment by xmrstickers** [comment]
initialize the wallet and unlock it through the 9053 admin panel to avoid this issue if you are experiencing it post 4.0.13
